web_src/css/base.css

```diff
@@ -2232,6 +2238,16 @@ a.ui.active.label:hover {
   border-left: none;
 }
 
+.ui.button.button-link {
+  background: transparent;
+  border: none;
+  color: inherit;
```

Comment:
What we should also think about is whether we set `margin: inherit;`
If I've seen correctly, buttons get margin by default otherwise while links don't.

Reply:
Indeed, we should undo any conflicting button styles here.

Reply:
On the other hand, I think we do want to retain size stuff so the "button" takes a area similar to what a regular button takes.